charts/sysdig-deploy/scripts/migrate_values.py

```diff
@@ -0,0 +1,107 @@
+import copy
+import sys
+import yaml
+
+
+## path mappings for values to be extracted to new paths
+VALUES_MAPPINGS = [
+    # globals
+    ("clusterName",                     "global.clusterConfig.name"),
+    ("sysdig.accessKey",                "global.sysdig.accessKey"),
+    ("sysdig.existingAccessKeySecret",  "global.sysdig.accessKeySecret"),
+    ("image.registry",                  "global.image.registry"),
+    # agent-only
+    ("auditLog",                        "agent.auditLog"),
+    ("collectorSettings",               "agent.collectorSettings"),
+    ("daemonset",                       "agent.daemonset"),
+    ("ebpf",                            "agent.ebpf"),
+    ("extraVolumes",                    "agent.extraVolumes"),
+    ("gke",                             "agent.gke"),
+    ("image.repository",                "agent.image.repository"),
+    ("image.tag",                       "agent.image.tag"),
+    ("image.pullPolicy",                "agent.image.pullPolicy"),
+    ("leaderelection",                  "agent.leaderelection"),
+    ("priorityClassName",               "agent.priorityClassName"),
+    ("prometheus",                      "agent.prometheus"),
+    ("proxy",                           "agent.proxy"),
+    ("psp",                             "agent.psp"),
+    ("rbac",                            "agent.rbac"),
+    ("resourceProfile",                 "agent.resourceProfile"),
+    ("scc",                             "agent.scc"),
+    ("secure",                          "agent.secure"),
+    ("serviceAccount",                  "agent.serviceAccount"),
+    ("slim",                            "agent.slim"),
+    ("sysdig.disableCaptures",          "agent.sysdig.disableCaptures"),
+    ("sysdig.settings",                 "agent.sysdig.settings"),
+    ("timezone",                        "agent.timezone"),
+    ("tolerations",                     "agent.tolerations"),
+    # node-analyzer-only
+    ("gke",                             "nodeAnalyzer.gke"),
```

Comment:
gke is being mapped to both agent on Line 19 and nodeAnalyzer on Line 39. I feel this might just pick the first value which is agent. gke every time when run in a for loop. Maybe I'm wrong?

Reply:
since it loops through the whole list on L81, I'm not sure why it should only pick the first. do you see something that in the code that might drop it?

Reply:
correction : LGTM